examples/notebooks/stationarity_detrending_0.ipynb

```diff
@@ -0,0 +1,304 @@
+   "metadata": {},
+   "source": [
+    "The KPSS tests gives the following results – test statistic, p value and the critical value at 1%, 5% , and 10% confidence intervals.\n",
+    "\n",
+    "KPSS test is now applied on the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kpss_test(sunspots['SUNACTIVITY'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Critical value of p is assumed to be 0.05. Based upon the p-value, the null hypothesis can not be rejected. Hence, the series is stationary as per the KPSS test. \n",
+    "\n",
+    "It is always better to apply both the tests, so that it can be ensured that the series is truly stationary. Possible outcomes of applying these stationary tests are as follows:\n",
+    "\n",
+    "Case 1: Both tests conclude that the series is not stationary - The series is not stationary  \n",
+    "Case 2: Both tests conclude that the series is stationary - The series is stationary  \n",
+    "Case 3: KPSS indicates stationarity and ADF indicates non-stationarity - The series is trend stationary. Trend needs to be removed to make series strict stationary  \n",
+    "Case 4: KPSS indicates non-stationarity and ADF indicates stationarity - The series is difference stationary. Differencing is to be used to make series stationary  \n",
+    "\n",
+    "Here, due to the differcne in the results from ADF test and KPSS test, it can be inferred that the series is trend stationary and not strict stationary. The series can be detrended by differencing or by model fitting."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Detrending by Differencing\n",
+    "\n",
+    "It is one of the simplest methods for detrending a time series. A new series is constructed where the value at the current time step is calculated as the difference between the original observation and the observation at the previous time step.\n",
+    "\n",
+    "Differencing is applied on the data and the result is plotted."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sunspots['SUNACTIVITY_diff'] = sunspots['SUNACTIVITY'] - sunspots['SUNACTIVITY'].shift(1)\n",
+    "sunspots['SUNACTIVITY_diff'].dropna().plot(figsize=(12,8))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "ADF test is now applied on these detrended values and stationarity is checked."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "adf_test(sunspots['SUNACTIVITY_diff'].dropna())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Based upon the p-value of ADF test, the null hypothesis can be rejected. Hence, the series is strict stationary now."
```

Comment:
Maybe "there is evidence in favor of rejecting the null hypothesis in favor of the alternative "

Reply:
alright